doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2032,36 +2078,39 @@ Response:
         [
           {
             "id": "abcdefg",
-            "subject": {
+            "description": "Example Context Source",
+            "dataProvided": {
               "entities": [
                 {
                   "id": "Bcn_Welt",
                   "type": "Room"
                 }
               ],
-              "attributes": [
+              "attrs": [
                 "temperature"
               ]
             },
-            "callback": "http://weather.example.com/ngsi",
-            "metadata": {
-              "providingService": {
-                 "value": "weather.example.com",
-                 "type": "Text"
-               },
-              "providingAuthority": {
-                 "value": "AEMET - Spain",
-                 "type": "Text"
-               }
+            "provider": {
+              "http": {
+                "url": "http://contextsource.example.org/v2/entities"
```

Comment:
Note 

> The endpoint must *not* include the protocol specific part (for instance `/v2/entities`)

Thus, I understand examples (this one an all the others in the .apib) should remove the `/v2/entities` part)

Reply:
fixed 78ce71dfa1b2c6952018ede3a9ef640c65abc1dd